tests/unit/ai/review/test_orchestrator.py

```
@@ -216,3 +216,45 @@ def test_run_review_depth2_calls_provider_twice() -> None:
+            run_review(
+                context,
+                provider=provider,
+                ai_config=AIConfig(enabled=True),
+                depth=1,
+                checklist_items=[],
+                checklist_text="1. [logic-bug] Example?",
+                classifications=[],
+                progress=progress,
+            )
+        except RuntimeError:
+            pass
```

Ruff (0.15.20)

[warning] Use `contextlib.suppress(RuntimeError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(RuntimeError): ...`

(SIM105)